code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -520,6 +529,77 @@ the implant may become unstable and either pre-maturely inject the subject or si
 /obj/item/weapon/implant/compressed/islegal()
 	return 0
 
+
+
 /obj/item/weapon/implant/cortical
 	name = "cortical stack"
 	desc = "A fist-sized mass of biocircuits and chips."
+
+
+
+/obj/item/weapon/implant/peace
+	name = "pax implant"
+	desc = "A bean-shaped implant with a single embossed word - PAX - on it."
+	var/imp_alive = 0
+	var/imp_msg_debounce = 0
+	var/imp_data = {"
+<b>Implant Specifications:</b><BR>
+<b>Name:</b> Pax Implant<BR>
+<b>Manufacturer:</b> Ouroboros Medical<BR>
+<b>Effect:</b> Makes the host incapable of committing violent acts.
+<b>Important Notes:</b> Effect accomplished via a chemical secreted by the implant. This chemical is neutralized by 15u or greater of Methylin.<BR>
+<b>Life:</b> Sustained as long as it remains within a host. Survives on the host's nutrition. Dies upon removal.<BR>
+"}
+
+/obj/item/weapon/implant/peace/meltdown()
+	for (var/mob/msg_viewer in viewers(src, null))
```

Comment:
This is the old way of doing visible_message. Use visible_message unless you've got a good reason to use this.